pkg/testutil/testutil_linux.go

```diff
@@ -25,6 +25,7 @@ func mirrorOf(s string) string {
 
 var (
 	AlpineImage                 = mirrorOf("alpine:3.13")
+	AmazonLinux2Image           = "public.ecr.aws/amazonlinux/amazonlinux:2"
```

Comment:
why is a new image needed here?

Reply:
I need to use curl in tests but alpine doesn't have it and AL2 has it. Let me know if you have suggestions for better image choice.

Reply:
I see. I'm okay with that. The [`nginx` image](https://github.com/containerd/nerdctl/blob/8a3331729b0d43fd51cec8aaaa5f45238ae05749/pkg/testutil/testutil_linux.go#L28) should have `curl`, can you try that?

Reply:
That works. Changed it to.